backend/scripts/dataset-import/.gitignore

```diff
@@ -1,3 +1,4 @@
 /metadata
 /public-dataset
 /dump.sql
+/venv-import-dataset
```

Comment:
Side note: we recently merged a change related to the dev-env (#365), to include a broader database. 
Hopefully it will make future development easier, if you are using Docker to instantiate a local environment. Some data related to videos and scores are provided out-of-the-box.

Reply:
Ok, thanks, that may be very useful.